src/stylist/__main__.py

```diff
@@ -8,8 +8,6 @@
 Tool for checking code style.
 '''
 
-from __future__ import absolute_import, division, print_function
```

Comment:
See my previous comment about dropping v2.7 support. I think we should make that a separate change.